RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,570 @@
+          externalDocs:
+            url: TBD
+          enum: 
+          - DURATION
+          - LIMIT
+          - RATE
+        capacity_value:
+          type: number
+          description: |-
+            The number of flights allowed without exceeding the capacity limit.  This
+            number represents the maximum capacity for the limit type. 
+          minimum: 0
+    ResourceCapacity:
+      required:
+      - capacity_limit_type
+      - capacity_property_array
+      properties:
+        capacity_limit_type:
+          $ref: "#/components/schemas/CapacityLimitType"
+        capacity_property_array:
+          minItems: 1
+          maxItems: 2
+          type: array
+          description: |-
+            Describes the properties of the capacity requirements defined for the resource.
+    TemporaryResourceCapacity:
+      required: 
+        - id
+        - resource_capacity
+        - capacity_start_time
+        # TODO: How do we delete a resource capacity that is no longer wanted?  
+        # We could make the capacity and start time optional and if not provided, the temp 
+        # capacity is deleted? 
+      description: |-
+        A temporary resource capacity which overrides the default Resource Capacity. 
+      properties:
+        id:
+          description: |- 
+            A unique identifier for this temporary change to the default resource capacity.  Included
+            to allow for future updates (ex. adding an end time) to an existing temporary capacity change. 
+          $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/EntityID'
+        resource_capacity:
+          $ref: "#/components/schemas/ResourceCapacity"
+        capacity_start_time:
+          description: The time at which the updated resource capacity begins. 
+          anyOf:
+          - $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Time'
+        capacity_end_time:
+          description: The time at which the updated resource capacity ends, if known. 
+            If not provided, the temporary capacity continues indefinitely. 
+          anyOf:
+          - $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Time'
+    ActiveResourceCapacity:
+      required: 
+        - resource_capacity
+      description: |-
+        The current resource capacity requirement in place. This may be a default or a 
+        temporary capacity requirement. 
+      properties:
+        resource_capacity:
+          $ref: "#/components/schemas/ResourceCapacity"
+        capacity_end_time:
+          description: The time at which the updated resource capacity ends, if known. 
+            If not provided, the current capacity continues indefinitely. 
+          anyOf:
+          - $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Time'
+    # TODO: Do we want to include some subset of future capacities within the resource details?
```

Comment:
I think we do, yes, as available

Reply:
What do you think about just including the next temporary capacity, if any exists?  That way from status you know the capacity right now and the next change that is expected to capacity.  I think we should include all future capacities in the capacities, so the 1 next temporary would be duplicated in both places.  Thoughts?

Reply:
Ok, I think I understand now. As you have it now, resource details only include the active (current) capacity. You would only get the future capacities when they are PUT to you by the RIS because you have a subscription. But it may be good to receive at least some of them (like the next capacity) when you get the resource details. Let's see what other think.